frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -94,13 +109,34 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex'
 import emoticonsMixins from './EmoticonsMixins.js'
+import Avatar from '@components/Avatar.vue'
 import Tooltip from '@components/Tooltip.vue'
+import { makeMentionFromUsername } from '@model/contracts/chatroom.js'
+
+const caretKeyCodes = {
+  ArrowLeft: 37,
+  ArrowUp: 38,
+  ArrowRight: 39,
+  ArrowDown: 40,
+  Esc: 27,
+  End: 35,
+  Home: 36
+}
```

Comment:
Should this also include `PageUp` and `PageDown`?

Reply:
No. Slack works that way. When we type `@` in the textarea and press `PageUp` or `PageDown`, the popup doesn't disappear.